<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrelationBayesianPairsForm</class>
 <widget class="QWidget" name="CorrelationBayesianPairsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>626</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="topWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="1" rowspan="4">
       <widget class="AvailableFieldsListView" name="availableFields">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3" rowspan="4">
       <widget class="BoundPairsTable" name="pairs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2" rowspan="2">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="AssignButton" name="assignButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="options" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>13</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="4" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="GroupBox" name="groupPrior">
           <property name="title">
            <string>Prior</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_11">
            <property name="topMargin">
             <number>6</number>
            </property>
            <item row="0" column="1">
             <widget class="BoundTextBox" name="priorWidth">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Stretched beta prior width</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="BoundGroupBox" name="hypothesis">
           <property name="title">
            <string>Hypothesis</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="2" column="0">
             <widget class="QRadioButton" name="_3oneTailedLessThan">
              <property name="text">
               <string>Correlated negatively</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="_2oneTailedGreaterThan">
              <property name="text">
               <string>Correlated positively</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="_1twoTailed">
              <property name="text">
               <string>Correlated</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="BoundGroupBox" name="bayesFactorType">
           <property name="title">
            <string>Bayes Factor</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="1" column="0">
             <widget class="QRadioButton" name="_2">
              <property name="text">
               <string>BF₀₁</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="_1">
              <property name="text">
               <string>BF₁₀</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="_3">
              <property name="text">
               <string>Log(BF₁₀)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="BoundGroupBox" name="missingValues">
           <property name="title">
            <string>Missing Values</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="_1excludeAnalysisByAnalysis">
              <property name="text">
               <string>Exclude cases analysis by analysis</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="_2excludeListwise">
              <property name="text">
               <string>Exclude cases listwise</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="GroupBox" name="plotsGroup">
           <property name="title">
            <string>Plots</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0" colspan="3">
             <widget class="BoundCheckBox" name="plotPriorAndPosterior">
              <property name="text">
               <string>Prior and posterior</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="3">
             <widget class="BoundCheckBox" name="plotSequentialAnalysis">
              <property name="text">
               <string>Sequential analysis</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="BoundCheckBox" name="plotBayesFactorRobustness">
              <property name="text">
               <string>Bayes factor robustness check</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="2">
             <widget class="BoundCheckBox" name="plotSequentialAnalysisRobustness">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Robustness check</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="BoundCheckBox" name="plotPriorAndPosteriorAdditionalInfo">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Additional info</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QWidget" name="widget_5" native="true">
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="BoundCheckBox" name="plotScatter">
              <property name="text">
               <string>Scatterplot</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="BoundGroupBox" name="corcoefficient">
        <property name="title">
         <string>Correlation Coefficient</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>Pearson's rho</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>Kendall's tau-b</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="additionalStatsContainer" native="true">
        <widget class="BoundCheckBox" name="credibleInterval">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>252</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Credible intervals</string>
         </property>
        </widget>
        <widget class="QWidget" name="ciValueContainer" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>241</width>
           <height>21</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget_9" native="true">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="BoundTextBox" name="ciValue">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Interval</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AssignButton</class>
   <extends>QPushButton</extends>
   <header>widgets/assignbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AvailableFieldsListView</class>
   <extends>QListView</extends>
   <header>widgets/availablefieldslistview.h</header>
  </customwidget>
  <customwidget>
   <class>GroupBox</class>
   <extends>QGroupBox</extends>
   <header>widgets/groupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundTextBox</class>
   <extends>QLineEdit</extends>
   <header>widgets/boundtextbox.h</header>
  </customwidget>
  <customwidget>
   <class>BoundGroupBox</class>
   <extends>QGroupBox</extends>
   <header>widgets/boundgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundCheckBox</class>
   <extends>QCheckBox</extends>
   <header>widgets/boundcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>BoundPairsTable</class>
   <extends>QTableView</extends>
   <header>widgets/boundpairstable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>plotSequentialAnalysis</sender>
   <signal>toggled(bool)</signal>
   <receiver>plotSequentialAnalysisRobustness</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plotPriorAndPosterior</sender>
   <signal>toggled(bool)</signal>
   <receiver>plotPriorAndPosteriorAdditionalInfo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>credibleInterval</sender>
   <signal>toggled(bool)</signal>
   <receiver>ciValueContainer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
